pkg/networkserver/grpc_gsns.go

```diff
@@ -633,7 +633,7 @@ func (ns *NetworkServer) handleUplink(ctx context.Context, up *ttnpb.UplinkMessa
 				case ttnpb.CID_LINK_ADR:
 					pld := cmd.GetLinkADRAns()
 					dupCount := 0
-					if stored.MACState.LoRaWANVersion == ttnpb.MAC_V1_0_2 {
+					if stored.MACState.LoRaWANVersion.Compare(ttnpb.MAC_V1_0_2) >= 0 {
```

Comment:
Assuming here that atomic application of multiple `LinkAdrReq` is enforced by 1.0.2 or higher.

Not sure why the previous check was exactly on 1.0.2; seems wrong.

Reply:
This was correct behavior. There must be no duplicates in 1.1+
![2019-04-15-11:16:33-screenshot](https://user-images.githubusercontent.com/12877905/56121256-08324980-5f70-11e9-89bf-ca491dd6b99b.png)

The check should be `1.0.2 <= ver < 1.1` for proper 1.0.X support

Reply:
Ok, please file issue